z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-compute-nodes/tasks/main.yaml

```diff
@@ -83,4 +82,34 @@
     terminate_volume: True
   with_indexed_items: "{{ [os_compute_server_name] * os_compute_nodes_number }}"
   when:
-  - disk_type == "scsi"
\ No newline at end of file
+  - disk_type == "scsi"
+  - use_storage_provider == false
+
+- name: 'Create BFV compute volume from backend storage provider'
+  os_volume:
+    state: present
+    name: "{{ item.1 }}-{{ item.0 }}-bfv"
```

Comment:
1. name -> display_name?
2. -bfv -> -root

Reply:
display_name was ICIC using name, in our OCP playbook we use name.